common/text/arial-labels.ts

```diff
@@ -5,3 +5,5 @@ export const searchFilterCheckBox = (label: string): string => {
 export const searchFormInputCatalogue = 'Search the catalogue';
 export const searchFormInputImage = 'Search for Images';
 export const searchFilterCloseButton = 'Close Filter';
+
+export const searchMoreFiltersLanguages = 'Filter by languages';
```

Comment:
can't see where this is used - also bit confused by the file name

Reply:
just a typo, presume it's meant to be aria-labels

Reply:
yeah not using this .. will remove